launcher/src/main/java/org/apache/spark/launcher/CommandBuilderUtils.java

```diff
@@ -334,6 +334,18 @@ static void addPermGenSizeOpt(List<String> cmd) {
   }
 
   /**
+   * Gets the OutOfMemoryError option for Spark if the user hasn't set it.
+   */
+  public static void addOutOfMemoryErrorArgument(List<String> cmd) {
```

Comment:
Sorry last question -- why does this method need to be in this class? it's not used, it seems, except from YarnSparkHadoopUtil